src/Utility/Text.php

```diff
@@ -215,7 +215,7 @@ public static function insert(string $str, array $data, array $options = []): st
 
         $dataKeys = array_keys($data);
         $hashKeys = array_map(
-            fn($str) => hash('xxh128', $str),
+            fn(int|string $str) => hash('xxh128', $str),
```

Comment:
```suggestion
            fn(int|string $str) => hash('xxh128', (string)$str),
```

2nd arg of `hash()` only accepts string